proto/elys/margin/proposals.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+package elys.margin;
+
+import "elys/margin/params.proto";
+
+option go_package = "github.com/elys-network/elys/x/margin/types";
+
+message ProposalUpdateParams {
+    string title = 1;
+    string description = 2;
+    Params params = 3;
+}
+
+message ProposalUpdatePools {
+    string title = 1;
+    string description = 2;
+    repeated string pools = 4;
+    repeated string closed_pools = 5;
+}
+
+message ProposalWhitelist {
+    string title = 1;
+    string description = 2;
+    string whitelisted_address = 3;
+}
+
+message ProposalDewhitelist {
+    string title = 1;
+    string description = 2;
+    string whitelisted_address = 3;
+}
+
+message ProposalAdminCloseAll {
```

Comment:
From Dream:
What is this admin related parameters. There should be NONE. The protocol shud handle everything.

Reply:
Yep, it got removed.